frontend/ext/varfish-api/src/lib/types.gen.ts

```
@@ -3394,62 +3416,66 @@ export type CasesImportApiCaseImportActionListCreateListData = {
+export type CasesImportApiCaseImportActionPartialUpdateError = unknown;
 
-export type CasesImportApiCaseImportActionRetrieveUpdateDestroyDestroyData = {
+export type CasesImportApiCaseImportActionDestroyData = {
     path: {
         caseimportaction: string;
+        project: string;
     };
 };
 
-export type CasesImportApiCaseImportActionRetrieveUpdateDestroyDestroyResponse = (void);
+export type CasesImportApiCaseImportActionDestroyResponse = (void);
```

Biome

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)